storm-core/src/jvm/org/apache/storm/messaging/netty/KerberosSaslNettyClient.java

```diff
@@ -84,7 +83,7 @@ public KerberosSaslNettyClient(Map storm_conf, String jaas_section) {
             Configuration.setConfiguration(login_conf);
             //now login
             LOG.debug("Trying to login.");
-            Login login = new Login(jaas_section, ch);
+            ZookeeperSaslLogin login = new ZookeeperSaslLogin(jaas_section, ch);
```

Comment:
I don't really like Zookeeper being a part of this classes name.  I also don't really like Sasl in there, because even though we use it for Sasl, there is nothing Sasl specific in it.